common/src/main/java/com/sequenceiq/cloudbreak/rotation/secret/type/FreeIpaSecretType.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.cloudbreak.rotation.secret.type;
+
+import static com.sequenceiq.cloudbreak.rotation.secret.step.CommonSecretRotationStep.SALT_BOOT;
+import static com.sequenceiq.cloudbreak.rotation.secret.step.CommonSecretRotationStep.USER_DATA;
+import static com.sequenceiq.cloudbreak.rotation.secret.step.CommonSecretRotationStep.VAULT;
+
+import java.util.List;
+
+import com.sequenceiq.cloudbreak.rotation.secret.SecretType;
+import com.sequenceiq.cloudbreak.rotation.secret.step.SecretRotationStep;
+
+public enum FreeIpaSecretType implements SecretType {
```

Comment:
as discussed, pls move this to freeipa-api to separate service related secret types